db/migrate/20230906165200_migrate_imported_variants_data.rb

```diff
@@ -0,0 +1,181 @@
+class MigrateImportedVariantsData < ActiveRecord::Migration[7.0]
+  class OptionValueNamer
```

Comment:
I'm guessing good_migrations forced you to copy this here. It looks strange because it's so large, but I guess it's the safest option 👍